code/_onclick/hud/alert.dm

```diff
@@ -497,7 +495,7 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 
 /atom/movable/screen/alert/cancharge
 	name = "Charge Ready"
-	desc = "You are ready to charge at a location!"
+	desc = "You are ready to charge to a location!"
```

Comment:
If we're being pendantic, technically "to a location" implies that you will reach the location, rather than that you are trying to reach a location, as "at a location" and "towards a location" imply.